src/components/Project.js

```diff
@@ -19,11 +20,18 @@ class Project extends Component {
 
   handleClick() {
     GoogleAnalytics.trackEvent('view', this.props.project.display_name)
+    if (this.props.project.redirect) {
```

Comment:
Look into using `.shape` when setting propTypes for items like objects. It's a way to further define ambiguous props like objects, so you could then tell it to expect a `redirect` tag. We've recently started doing this in PFE. This is probably fine for now.